harper-core/src/word_metadata.rs

```diff
@@ -113,6 +113,41 @@ impl WordMetadata {
         )
     }
 
+    /// Checks if the word is a homophone
+    /// This is based off of metadata, so it is not guaranteed to be correct
+    pub fn is_homophone(&self) -> bool {
+        [
+            matches!(
+                self.noun,
+                Some(NounData {
+                    is_proper: None,
+                    is_plural: None,
+                    is_possessive: None,
+                    is_pronoun: None
+                })
+            ),
+            matches!(
+                self.verb,
+                Some(VerbData {
+                    is_linking: None,
+                    tense: None
```

Comment:
It feels like implementation is likely to result in an inconsistency with the model for `WordMetadata`. Is there a way to make this throw an error if we modify `WordMetadata` but don't update the code here?

Reply:
Hmm, not sure about that. I can't think of any way.

Reply:
I found a way. We're good here.